MarsCompetition/MarsCompetition/Test/CertificationTest.cs

```diff
@@ -0,0 +1,92 @@
+﻿using AventStack.ExtentReports;
+using MarsCompetition.Pages;
+using MarsCompetition.TestModel;
+using MarsCompetition.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MarsCompetition.Test
+{
+    public class CertificationTest : CommonDriver
+    {
+#pragma warning disable CS8618
+
+        private CertificationPage certificationPageObj = new CertificationPage();
+        private ExtentReports extent;
+        [Test, Order(1)]
+        public void AddCertification_Test()
+        {
+            CommonDriver driver = new CommonDriver();
+            driver.InitializeExtentReports();
+            string testName = "AddCertification";
+            driver.SetupTest(testName);
+            string sFile = "AddCerPosdata.json";
+            // Read test data from the JSON file using JsonHelper
+            List<CertificationTestModel> AddCerPosdata = Jsonhelper.ReadTestDataFromJson<CertificationTestModel>(sFile);
+            Console.WriteLine(AddCerPosdata.ToString());
```

Comment:
All your tests are missing how it handles state the state is say that you have a test that is adding a language english, the second time you run this test you should delete any existing languages so that the state is reset, this will guarantee that your test does not produce false negatives and passes always

correct me if you have done this but I could not see this

Reply:
if you have not done this please do this for all tests